doc/catalog-format-spec.md

```diff
@@ -455,6 +455,12 @@ static, i.e. they cannot depend on the context.
         several crates from the same repository (sometimes referred to as a
         *monorepo*).
 
+      - `binary`: optional (defauts to false) boolean used to design the origin
```

Comment:
Maybe we can disable implicit detection then?

Reply:
This will require editing all binary releases in the index. We are due for an index version bump with the 2.0 release, so that would be a good moment.

Reply:
It would make things more clear for everyone I think.